server/server.js

```diff
@@ -8,3 +8,5 @@ app.set('port', port);
 const server = http.createServer(app);
 server.listen(port);
 console.log('Connected on port ', port);
+
```

Comment:
be consistent in the use of ES6 features use export default server instead of module.exports